src/kaltura-player.js

```diff
@@ -510,6 +516,33 @@ class KalturaPlayer extends FakeEventTarget {
   get Error(): typeof Error {
     return this._localPlayer.Error;
   }
+
+  /**
+   * set the share config
+   * @returns {void}
+   * @private
+   */
+  _maybeSetEmbedConfig(): void {
```

Comment:
Could you explain what you do in this method?
from my understanding, you take values from share config and merge them back to the same room.